payments-core/src/main/java/com/stripe/android/googlepaylauncher/GooglePayPaymentMethodLauncherViewModel.kt

```diff
@@ -134,18 +134,20 @@ internal class GooglePayPaymentMethodLauncherViewModel(
             val config = PaymentConfiguration.getInstance(application)
             val publishableKey = config.publishableKey
             val stripeAccountId = config.stripeAccountId
+            val productUsageTokens = setOf(GooglePayLauncher.PRODUCT_USAGE)
```

Comment:
Should this be GooglePayPaymetMethod, or there is no separate product usage for that?

Reply:
There's no separate product, both are "GooglePayLauncher".